### PR TITLE
Yiz/csv static tables

### PR DESCRIPTION
**Pull request by sleepyzzzzzz**
## Changes made in this PR
generate CSV files for static tables and load into Postgres

## Implementation Checklist
Please check following check list to ensure the tasks are considered and completed in this branch.

#### API Design
- [ ] Defined gRPC API in proto file
- [ ] regenerated go code using `make proto_generate` command

#### API implementation
- [ ] Implemented API in corresponding gRPC server
- [ ] Added tests in gRPC layer

#### Domain Service and DB query Implementation
- [ ] Added new query in dbstore layer
- [ ] Added tests in db store layer
                                                                                       
- [ ] Implemented functionality in Domain Service Layer
- [ ] Added test in Domain Service Layer

- [ ] Created a new branch in paramount-api, with comments there indicating the functionality in Ruby that's re-implemented in Golang

#### Postgres Layer - identify and potentially add db migration to create index for fast querying.
- [ ] Verified and potentially created index for the newly added query.

### CODE REVIEW

File: migrations/csv/pdm_account_type_to_permissions.csv [hunk 1]
```diff
@@ -0,0 +1,295 @@
+id,created_at,updated_at,account_type_id,permission_id
```
Comment (lhan-the-real) [inline migrations/csv/pdm_account_type_to_permissions.csv:1]: Are we going to merge account_type_to_permissions table to account_type table?

File: migrations/csv/pdm_feature_to_permissions.csv [hunk 2]
```diff
@@ -0,0 +1,35 @@
+id,created_at,updated_at,feature_id,permission_id
```
Comment (lhan-the-real) [inline migrations/csv/pdm_feature_to_permissions.csv:1]: are we going to merge this table to features table?

File: compose.yaml [hunk 3]
```diff
@@ -51,6 +68,8 @@ services:
       interval: 5s
       timeout: 3s
       retries: 5
+    volumes_from:
```
Comment (lhan-the-real) [inline compose.yaml:71]: Awesome!!!!!

File: migrations/csv/pdm_account_type_to_permissions.csv (hunk 1 above)
Comment (sleepyzzzzzz) [reply migrations/csv/pdm_account_type_to_permissions.csv:1]: already to merged forgot to deleted

File: migrations/csv/pdm_feature_to_permissions.csv (hunk 2 above)
Comment (sleepyzzzzzz) [reply migrations/csv/pdm_feature_to_permissions.csv:1]: already merged
this csv just an export data for this association table ref

File: migrations/csv/pdm_account_type_to_permissions.csv (hunk 1 above)
Comment (lhan-the-real) [reply migrations/csv/pdm_account_type_to_permissions.csv:1]: If we've already merged the table, does the original account_type_to_permissions table data saved into account_type table?

Comment (sleepyzzzzzz) [reply migrations/csv/pdm_account_type_to_permissions.csv:1]: yes
Screenshot 2022-03-11 092330
Screenshot 2022-03-11 092402

**Pull request merged**